src/migrations/00012-rename-root-user.sql

```diff
@@ -1 +1,11 @@
+CREATE TABLE IF NOT EXISTS "user-has-public key" (
+	"created at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
+,	"user" INTEGER NOT NULL
+,	"public key" TEXT NOT NULL
+,	"id" SERIAL NOT NULL PRIMARY KEY
+,	"title" VARCHAR(255) NOT NULL
+,	FOREIGN KEY ("user") REFERENCES "user" ("id")
+,	UNIQUE("user", "public key")
+);
```

Comment:
@Page- this is actually interesting -- this is needed because this table is referenced in the `modified-at` migration. We haven't released an open-balena version that would cause this migration to run, and have the table automatically created, so Pine will execute this migration and the `modified-at` one in the same "transaction" and fail. This is interesting because there's a gotcha here we haven't been taking into account: we may introduce a new table and in later migrations we assume this table exists because there's been a deployment.